hack/verify_clientset.sh

```diff
@@ -38,7 +38,7 @@ make clientset
 
 echo "diffing ${DIFFROOT} against freshly generated codegen"
 ret=0
-diff -Naupr "${DIFFROOT}" "${TMP_DIFFROOT}" || ret=$?
+diff -Naupr "${DIFFROOT}" "${TMP_DIFFROOT}" -x '*.bazel'|| ret=$?
```

Comment:
Some of the bazel files might just be named `BUILD`, should we add that in the exclusion here?